envs/ci/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+name: acih-ci
+
+services:
+  app-build: &app-build
+    build:
+      context: ../..  # path from the current file to the project root dir
+      dockerfile: envs/ci/Dockerfile  # path from the project root dir to the Dockerfile
+
+  mypy:
+    <<: *app-build
+    entrypoint: mypy
+
+  ruff:
+    <<: *app-build
+    entrypoint: ruff check src
```

Comment:
We need also to lint code in `tests` directory. And same with pylint.

But before that we need to make a quickfix, because tests have already been added in the separate story but those changes didn't include changes in linters configuration, README.md, etc.

So once that quickfix will be in place we need to add src directory to linters here, in this PR